MANUAL_GCP_DEPLOYMENT.md

```diff
@@ -0,0 +1,18 @@
+# Deployment Guide for membrane-backend
+
+This guide outlines the steps to deploy manually membrane-backend to Google Cloud Run.
+
+## Steps
+ 1. Setup your GCP environment. You can use the instructions [GCP Project Setup Guide](https://github.com/ai-cfia/devops/blob/main/gcp-setup-script/gcp-project-setup-guide.md)
+
+2.  You will need to add the "Secret Manager Secret Accessor" role to your service account
+
+3. Enable "Secret Manager" service on your GCP project.
+
+4. After running the script from the ReadMe file, you should have 4 keys on your project. If you have the public key(s) from your client(s), you can go ahead and delete the one you generated and replace it with the one(s) you have. In the "Secret Manager", create a new secret and for each key, give it the right name and upload it.
```

Comment:
why not link to the readme file? Be empathetic to your reader and provide him links so he doesn't have to search.

Reply:
my though process was the everyone that is to the deployment steps already went through the steps of the ReadMe file but I agree if I reference it it makes more sense to link it for convenient sake